qa/src/test/resources/io/camunda/migrator/bpmn/c8/timerDurationBoundaryEventProcess.bpmn

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?><bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:conversion="http://camunda.org/schema/conversion/1.0" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" conversion:converterVersion="0.12.4" exporter="Camunda Modeler" exporterVersion="5.24.0" expressionLanguage="http://www.w3.org/1999/XPath" id="Definitions_1twsy0k" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.6.0" targetNamespace="http://bpmn.io/schema/bpmn" typeLanguage="http://www.w3.org/2001/XMLSchema">
+  <bpmn:process id="timerDurationBoundaryEventProcessId" isClosed="false" isExecutable="true" name="timerDurationBoundaryEventProcess" processType="None">
+    <bpmn:extensionElements>
+      <conversion:message severity="INFO">Unused attribute 'historyTimeToLive' on 'process' is removed.</conversion:message>
+    </bpmn:extensionElements>
+    <bpmn:startEvent id="StartEvent_1" isInterrupting="true" name="start" parallelMultiple="false">
+      <bpmn:extensionElements>
+        <zeebe:executionListeners>
+          <zeebe:executionListener eventType="end" type="migrator" />
+        </zeebe:executionListeners>
+      </bpmn:extensionElements>
+      <bpmn:outgoing>Flow_10ucyxv</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:endEvent id="userTaskEndEventId" name="userTaskEnd">
+      <bpmn:incoming>Flow_1mv8nmm</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_1mv8nmm" sourceRef="userTaskId" targetRef="userTaskEndEventId"/>
+    <bpmn:userTask completionQuantity="1" id="userTaskId" implementation="##unspecified" isForCompensation="false" name="userTask" startQuantity="1">
+      <bpmn:extensionElements>
+        <zeebe:userTask/>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_10ucyxv</bpmn:incoming>
+      <bpmn:outgoing>Flow_1mv8nmm</bpmn:outgoing>
+    </bpmn:userTask>
+    <bpmn:boundaryEvent attachedToRef="userTaskId" cancelActivity="true" id="timerBoundaryEventId" name="timerBoundary" parallelMultiple="false">
+      <bpmn:outgoing>Flow_1ydkpis</bpmn:outgoing>
+      <bpmn:timerEventDefinition id="TimerEventDefinition_0n5gunm">
+        <bpmn:timeDuration xsi:type="bpmn:tFormalExpression">=if (leftoverDuration != null) then leftoverDuration else "P10D"</bpmn:timeDuration>
```

Comment:
❓ Is it possible to test what is this expression evaluated to in the test?

Reply:
Or adding an assert before 
```
    processTestContext.increaseTime(Duration.ofDays(2));
```

Reply:
> ❓ Is it possible to test what is this expression evaluated to in the test?

Yes, this is being tested by checking that the timer has fired as expected after the leftover time (2days) instead of the default time (10days), this way we know the expression was evaluated correctly

> Or adding an assert before
> processTestContext.increaseTime(Duration.ofDays(2));

What kind of assert do you mean?

Reply:
Ah, you mean like asserting the timer hasnt fired yet before the 2d increase?

Reply:
Yes, thanks!